pkgs/by-name/bi/bitbox-bridge/package.nix

```
@@ -7,7 +7,7 @@
   libudev-zero,
 }:
 
-rustPlatform.buildRustPackage (finalAttrs: {
+rustPlatform.buildRustPackage (finalAttrs: rec {
```

There is no need for `rec` while using `finalAttrs`

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

---

Nope, the rec is required for the src reference below.

---

`finalAttrs.src` does the same, and it is also not needed, you can operate on the `$src` directly without needing to reference its derivation